### PR TITLE
ISSD-1554 [Marketplace] Create General Header for Order Details

### PR DESCRIPTION
**Pull request by claricebrasil**

### CODE REVIEW

**Comment by claricebrasil** [comment]
ci:test:sf

**Review by eliasantosf — APPROVED** [review]

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import './OrderDetailsHeader.scss';
+import OrderDetailsStatusDescription from './OrderDetailsStatusDescription';
+
+interface OrderDetailsProps {
+	hasOrderDescription?: boolean;
+	hasOrderDetails?: boolean;
+	orderDetails?: Cart;
+	productCreatorAccount?: string;
+	productImage?: string;
+	productName?: string;
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx:17]:
```suggestion
	hasOrderDescription?: boolean;
	hasOrderDetails?: boolean;
	order?: Cart;
	productOwner?: string;
	image?: string;
	name?: string;
```

File: workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+}
+
+const OrderDetailsHeader = ({
+	hasOrderDescription = false,
+	hasOrderDetails = false,
+	orderDetails,
+	productCreatorAccount,
+	productImage,
+	productName,
```
Comment (kevenleone) [inline workspaces/liferay-marketplace-workspace/client-extensions/liferay-marketplace-custom-element/src/pages/PurchasedAppsDashboard/Apps/components/OrderDetailsHeader.tsx:26]:
```suggestion
	hasOrderDescription = false,
	hasOrderDetails = false,
	order,
	productOwner,
	image,
	name,
```

**Review by kevenleone — CHANGES_REQUESTED** [review]

**Comment by kevenleone** [comment]
Just starting reviewing :)

**Comment by claricebrasil** [comment]
resent: #143334

**Comment by claricebrasil** [comment]
resent: https://github.com/brianchandotcom/liferay-portal/pull/143334

**Pull request closed without merging**